### PR TITLE
Feature/history,like,dislike

### PR DESCRIPTION
**Pull request by thegeorgenikhil**
Users can now:
- view their watch history
- delete their watch history(single or all)
- like video
- remove a liked video

Added **single video page**

### CODE REVIEW

**Review by surajmore98 — APPROVED** [review]
good structured code.
Looks good to merge.

File: src/components/Loader/Loader.jsx [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
+export const Loader = () => {
   return (
     <div className="loader-container">
-      <CircleSpinner
-        size={1.2}
-        sizeUnit={"rem"}
-        color="#FFFFFF"
-      />
+      <CircleSpinner size={1.2} sizeUnit={"rem"} color="#FFFFFF" />
     </div>
```
Comment (Shah-Arsalan) [inline src/components/Loader/Loader.jsx:7]: color variable can be used

File: src/pages/VideoPage/VideoPage.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+              >
+                <MdOutlinePlaylistPlay />
+                Add To Playlist
+              </p>
+            </div>
+            <div className="flex items-center gap-2">
+              <img
+                className="channel-img"
+                src={videoInfo?.channelImgUrl}
+                alt=""
```
Comment (Shah-Arsalan) [inline src/pages/VideoPage/VideoPage.jsx]: You can add alt value

**Review by Shah-Arsalan — APPROVED** [review]
Code looks clean . Good use of variables for reducer . Overall folder structure is apt . You can merge the code

File: src/pages/VideoPage/VideoPage.jsx (hunk 2 above)
Comment (thegeorgenikhil) [reply src/pages/VideoPage/VideoPage.jsx]: fixed it

**Pull request merged**